operations/mimir-tests/test-consul-generated.yaml

```diff
@@ -1158,7 +1157,6 @@ spec:
     spec:
       containers:
       - args:
-        - -alertmanager-storage.backend=gcs
```

Comment:
Isn't missing the common storage config here?

Reply:
Yes, good catch. Fixed in https://github.com/grafana/mimir/commit/00d0f3f976850fb51de72eb1d0f45ba0da0b9d29 and committed the test in https://github.com/grafana/mimir/pull/3257/commits/654f06690d5ad3e70a0002d193a90a07f7663071